modules/nf-core/clair3/meta.yml

```
@@ -0,0 +1,118 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "clair3"
+description: germline small variant caller
+keywords:
+- sort
+- example
+- genomics
+tools:
+- "clair3":
+    description: "Clair3 is a small variant caller for long-reads. Compare to PEPPER
+      (r0.4), Clair3 (v0.1) shows a better SNP F1-score with ≤30-fold of ONT data
+      (precisionFDA Truth Challenge V2), and a better Indel F1-score, while runs generally
+      four times faster. Clair3 makes the best of both worlds of using pileup or full-alignment
+      as input for deep-learning based long-read small variant calling. Clair3 is
+      simple and modular for easy deployment and integration."
+    homepage: "https://github.com/HKU-BAL/Clair3"
+    documentation: "https://github.com/HKU-BAL/Clair3"
+    tool_dev_url: "https://github.com/HKU-BAL/Clair3"
+    doi: "10.1038/s43588-022-00387-x"
+    licence: ['BSD-3-clause']
+    identifier: biotools:clair3
+
+input:
+- - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
```

```suggestion
        e.g. `[ id:'sample1' ]`
```
Can we remove all the single_end:false if possible, as they aren't relevant here.